experiment_prototype/experiment_prototype.py

```diff
@@ -34,42 +34,30 @@
-    All other may differ.
-
-INTEGRATION
-    integration by integration interfacing (one
-    pulse_sequence of one exp_slice, then the next). exp_slice #1 and
-    exp_slice #2 must have same intt and intn. Integrations will
-    switch between one and the other until time is up/nave is
-    reached.
-
-PULSE
-    Simultaneous pulse_sequence interfacing, pulse by pulse
-    creates a single pulse_sequence. exp_slice A and B might have different
-    frequencies (stereo) and/or may have different pulse
-    length, tau_spacing, pulse_sequence, but must have the same integration
-    time. They must also have same len(scan), although they may
-    use different directions in scan. They must have the same
-    scan boundary if any. A time offset between the pulses
-    starting may be set (seq_timer in exp_slice). exp_slice A
-    and B will have integrations that run at the same time.
+Interfacing in this case refers to how two or more components are meant to be run together. The following types of interfacing are possible:
+
+1. SCAN.
+The scan by scan interfacing allows for slices to run a scan of one slice, followed by a scan of the second. The scan mode of interfacing typically means that the slice will cycle through all of its beams before switching to another slice.
+    There are no requirements for slices interfaced in this manner.
+
+2. INTTIME.
+This type of interfacing allows for one slice to run its integration period (also known as integration time or averaging period), before switching to another slice's integration period. This type of interface effectively creates an interleaving scan where the scans for multiple slices are run 'at the same time', by interleaving the integration times.
+    Slices which are interfaced in this manner must share:
+        - the same SCANBOUND value.
+
+3. INTEGRATION.
+Integration interfacing allows for pulse sequences defined in the slices to alternate between each other within a single integration period. It's important to note that data from a single slice is averaged only with other data from that slice. So in this case, the integration period is running two slices and can produce two averaged datasets, but the sequences (integrations) within the integration period are interleaved.
+    Slices which are interfaced in this manner must share:
+        - the same SCANBOUND value.
+        - the same INTT or INTN value.
+        - the same BEAM_ORDER length (scan length)
+
+4. PULSE.
+Pulse interfacing allows for pulse sequences to be run together concurrently. Slices will have their pulse sequences mixed and layered together so that the data transmits at the same time. For example, slices of different frequencies can be mixed simultaneously, and slices of different pulse sequences can also run together at the cost of having more blanked samples. When slices are interfaced in this way the radar is truly transmitting and receiving the slices simultaneously.
```

Comment:
I'm just wondering about using the term 'mixed' here. We are just adding the two waveforms together if their pulses overlap, and not really mixing them (in terms of RF mixing). Can we add this term to the list of terms to check usages of, and reword this, to portray that we are linearly summing the two pulse sequences?